sdk/networkcloud/Azure.ResourceManager.NetworkCloud/tests/Scenario/NetworkCloudClustersTests.cs

```diff
@@ -14,7 +17,9 @@ namespace Azure.ResourceManager.NetworkCloud.Tests.ScenarioTests
 {
     public class NetworkCloudClustersTests : NetworkCloudManagementTestBase
     {
-        public NetworkCloudClustersTests(bool isAsync, RecordedTestMode mode) : base(isAsync, mode) {}
+        public NetworkCloudClustersTests(bool isAsync, RecordedTestMode mode) : base(isAsync, mode) {
+            SanitizersToRemove.Add("AZSDK3402");
```

Comment:
This should be added in the same place we are adding the BodyKeySanitizer for ContainerUri

Reply:
moved to NetworkCloudManagementTestBase